src/neofs_testlib/cli/neofs_cli/shards.py

```diff
@@ -122,17 +121,15 @@ def list(
             json_mode: Print shard info as a JSON array.
             endpoint: Remote node address (as 'multiaddr' or '<host>:<port>').
             wallet: WIF (NEP-2) string or path to the wallet or binary key.
-            wallet_password: Wallet password.
 
         Returns:
             Command's result.
         """
-        return self._execute_with_password(
+        return self._execute(
```

Comment:
well, not sure i understand this PR

> This command doesn't require wallet password anymore

because of what? are we talking about `neofs-cli control shards list`?

Reply:
Yes.

Reply:
I misunderstood code a little, now - we provide a path to a wallet config file that contains password, so specifying password here is redundant and leads to an error due to an incorrect 'expect' here. I updated the whole file - removed password parameter everywhere.

Reply:
But where the config comes from? I don't see it in parameters. Also, commit message should have these details.

Reply:
And btw, the config should also have the wallet path inside, so maybe we still don't need it in parameters.

Reply:
Config comes from the time when we instantiate a CLI object to work with. It is hidden by parent class 'base' attributes. Currently it is done the same way in other commands. And a wallet is specified everywhere. If we don't need it actually - I can update the whole library then.

Reply:
Check the config itself, it should contain wallet path. If that's the case, then yes, we don't need to specify wallet again in parameters.

Reply:
No, it doesn't contain the wallet path and is dynamically generated during tests

Reply:
I've updated the commit message with the corresponding information about a config file usage